frontends/election-manager/src/components/export_logs_modal.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useContext, useEffect, useState } from 'react';
+import path from 'path';
+import { usbstick, throwIllegalValue } from '@votingworks/utils';
+
+import assert from 'assert';
+import { LogEventId } from '@votingworks/logging';
+import { AppContext } from '../contexts/app_context';
+import { Modal } from './modal';
+import { Button } from './button';
+import { Prose } from './prose';
+import { LinkButton } from './link_button';
+import { Loading } from './loading';
+import { MainChild } from './main';
+import { UsbImage } from './save_file_to_usb';
+
+const { UsbDriveStatus } = usbstick;
+
+const LOGS_ROOT_LOCATION = '/var/log';
+const LOG_NAME = 'vx-logs.log';
+
+export interface Props {
+  onClose: () => void;
+}
+
+enum ModalState {
+  ERROR = 'error',
+  SAVING = 'saving',
+  DONE = 'done',
+  INIT = 'init',
```

Comment:
Any reason for these to be shouty?

```suggestion
  Error = 'error',
  Saving = 'saving',
  Done = 'done',
  Init = 'init',
```